SiVL19/get_peir_gross.py

```diff
@@ -86,9 +87,12 @@ def split_images(images, keys, filename):
 				filename = image.get("alt")
 				image_src = image.get("src")
 				description = image.get("title").replace("\r\n", " ")
+				image_captions[filename] = description
+
+				tags_container = image_soup.find("div", {"id":"Tags"})
+				tags = [tag.string for tag in tags_container.findChildren("a")]
```

Comment:
Tag "gross" is present in all the images. Maybe we should not include it?

Reply:
Sounds like a pre-processing step.